weblate/checks/format.py

```diff
@@ -415,7 +414,10 @@ def get_description(self, check_obj):
         if results:
             errors.extend(self.format_result(results))
         if errors:
-            return mark_safe("<br />".join(escape(error) for error in errors))
+            return format_html_join(
+                format_html("<br />"), "{}",
+                ((error,) for error in errors)
```

Comment:
Not sure about this:

```suggestion
                (error for error in errors)
```